src/nat/front_ends/fastapi/fastapi_front_end_plugin.py

```
@@ -42,7 +57,33 @@ def get_worker_class_name(self) -> str:
+        from nat.front_ends.fastapi.job_store import JobStore
+
+        job_store = JobStore(scheduler_address=scheduler_address, db_url=db_url)
+
+        logger.info("Starting periodic cleanup of expired jobs every %d seconds", sleep_time_sec)
+        while True:
+            await asyncio.sleep(sleep_time_sec)
+
+            try:
+                await job_store.cleanup_expired_jobs()
+                logger.debug("Expired jobs cleaned up")
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)